### PR TITLE
Change log level from warn to info contextAvailable exception

### PR DESCRIPTION
**Pull request by AlvaroVega**
Log about "Wrong expression found .... over " is produced by contextAvailable of jexl plugin:

from https://github.com/telefonicaid/iotagent-node-lib/commit/d29e0060198355fde96153f31c5e7c8cdcf1d5f8

### CODE REVIEW

**Comment by AlvaroVega** [comment]
Maybe there was a misunderstanding with  https://github.com/telefonicaid/iotagent-node-lib/issues/1295

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Fix: change log level contextAvailable expression exception
+- Fix: remove mongo `DeprecationWarning: current Server Discovery an7d Monitoring engine is deprecated` by setting `useUnifiedTopology = true`
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:2]:
```suggestion
```

Duplicated

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- Fix: change log level contextAvailable expression exception
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: change log level contextAvailable expression exception (from WARN to INFO)

```

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -1,3 +1,5 @@
+- Fix: change log level contextAvailable expression exception (from WARN to INFO)
+
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:2]:
```suggestion
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**